mws/utils.py

```diff
@@ -151,6 +151,9 @@ def enumerate_param(param, values):
             MarketplaceIdList.Id.3: 4343
         }
     """
+    if not isinstance(values, (list, tuple, set)):
+        values = [values]
+
     if not any(values):
         # if not values -> returns ValueError
         return {}
```

Comment:
Seems fair, but note the next lines after this comment are identical to the new lines added:

https://github.com/python-amazon-mws/python-amazon-mws/blob/20415aaf4307b55bd9c206284767a633b1f8ec6d/mws/utils.py#L154-L159

Seems this just required some rearrangement of current code. Can you remove the second `isinstance` test, please?